programming/minibot-assembly-tutorial.md

```diff
@@ -0,0 +1,101 @@
+# Minibot Assembly
+### Instructions on how to build your own mini robot!
+#### How to program your minibot is on the following tutorial. 
+
+Creating a minibot is great for testing small projects. It's an easy change to try out new parts out and costs considerably less than a larger robot. 
+
+------------------------------
+### PARTS NEEDED:
```

Comment:
- Headings should not be in all caps
- Headings should not end with a colon
- This should not be a third-level heading, it should be a second-level heading